specification/schemaregistry/data-plane/Microsoft.EventHub/preview/2018-01-01-preview/schemaregistry.json

```diff
@@ -1,132 +1,279 @@
 {
   "swagger": "2.0",
   "info": {
-    "version": "2018-01-01-preview",
-    "title": "schemaregistry",
-    "description": "Description of the new service",
-    "x-ms-code-generation-settings": {
-      "name": "schemaregistryClient"
-    }
+    "title": "Azure Schema Registry",
+    "version": "2018-01-01-preview"
   },
-  "host": "management.azure.com",
-  "schemes": ["https"],
-  "consumes": ["application/json"],
-  "produces": ["application/json"],
-  "security": [
-    {
-      "azure_auth": ["user_impersonation"]
-    }
-  ],
-  "securityDefinitions": {
-    "azure_auth": {
-      "type": "oauth2",
-      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
-      "flow": "implicit",
-      "description": "Azure Active Directory OAuth2 Flow",
-      "scopes": {
-        "user_impersonation": "impersonate your user account"
+  "x-ms-parameterized-host": {
+    "hostTemplate": "{endpoint}/$schemagroups",
+    "useSchemePrefix": false,
```

Comment:
If you want the code generator to automatically add the `https` scheme you defined below, please remove this line.  Having `"useSchemePrefix": false` means you expect the caller to provide it.